scania-digital-design-system/sdds
# fix(cross-icon): improved svg file from figma

**Describe pull-request**  
New SVG file for Cross icon

**Solving issue**  
Fixes: DTS-1245

**How to test**  
1. Go to Icons Web Component
2. Select "Cross"
3. Compare this cross with one in the [main Storybook ](https://tegel-storybook.netlify.app/)
4. Additional: Compare X size in a banner component in this link and [main Storybook ](https://tegel-storybook.netlify.app/)
5. In both cases X should appear bigger

**Additional context**  
If correct, the icon will be uploaded to web fonts, aws, websites...
